src/components/android-settings-container.js

```diff
@@ -31,8 +33,8 @@ class AndroidSettingsContainer extends React.Component {
     state = {
         metadataSync: metadataSync,
         dataSync: dataSync,
-        numberSmsToSend: '',
-        numberSmsConfirmation: '',
+        numberSmsToSend: numberSmsToSend,
```

Comment:
You can use object shorthand in a lot of places in this file (probably elsewhere too)